api/internal/init.js

```diff
@@ -69,6 +74,24 @@ if ((globalThis.window || globalThis.self) === globalThis) {
   }
 }
 
+// webview only features
+if ((globalThis.window) === globalThis) {
+  globalThis.addEventListener('dragdropfiles', async (event) => {
+    const { files } = event.detail
+    const handles = []
+    for (const file of files) {
+      const stats = await fs.stat(file)
```

Comment:
Can we add a `typeof file === string` check?

Reply:
Done